README.md

```diff
@@ -34,15 +34,18 @@ regarding Chainlink social accounts, news, and networking.
 
 1. [Install Go 1.22](https://golang.org/doc/install), and add your GOPATH's [bin directory to your PATH](https://golang.org/doc/code.html#GOPATH)
    - Example Path for macOS `export PATH=$GOPATH/bin:$PATH` & `export GOPATH=/Users/$USER/go`
+   - Note that using Go 1.23, the current install at the above path, will result in 2 errors the first of which is `undefined: runtime_cyclesPerSecond`
```

Comment:
This is not normal. Can you elaborate?

Reply:
Which version of chainlink are you trying to build? I cannot reproduce this.

Reply:
It looks like the environment has affected the deps being used. Do you have a go.work file in scope?

Reply:
> It looks like the environment has affected the deps being used. Do you have a go.work file in scope?

I found go.work in ~/go/pkg/mod/google.golang.org/api@0.188.0. I don't know if that's in scope, not sure how I would figure that out.

Reply:
> Which version of chainlink are you trying to build? I cannot reproduce this.

I have a clone of /chainlink from last week, so I think I'm building the default version per the readme

Reply:
@jmank88 ping

Reply:
Is this still a problem? It shouldn't be happening on develop anymore